README.md

```diff
@@ -272,6 +272,10 @@ Your contributions and suggestions are heartily♥ welcome. (✿◕‿◕). Plea
 * [FuzzDB](https://github.com/fuzzdb-project/fuzzdb) - Dictionary of attack patterns and primitives for black-box application fault injection and resource discovery.
 * [EyeWitness](https://github.com/ChrisTruncer/EyeWitness) - Tool to take screenshots of websites, provide some server header info, and identify default credentials if possible.
 * [webscreenshot](https://github.com/maaaaz/webscreenshot) - A simple script to take screenshots of list of websites.
+* [Pwnwiki](http://pwnwiki.io) - Collection of TTP (tools, tactics and procedures) for what to do once access has been gained.
+* [Hackarmoury](http://hackarmoury.com.com) - Large repository hosting common information security tools. Helpful for uploading tools during post-exploitation phase.
+* [RedSnarf](https://github.com/nccgroup/redsnarf) - RedSnarf is a pen-testing/red-teaming tool for Windows enviroments.
+[Empire](https://github.com/gold1029/Empire) - Empire is a post-exploitation framework that includes a pure-PowerShell2.0 Windows agent, and a pure Python 2.6/2.7 Linux/OS X agent. (http://www.powershellempire.com) (https://github.com/EmpireProject/Empire)
```

Comment:
@emilyanncr would you be willing to fix this line? Looks like that would not make into becoming a list item in markdown